http4s-server/src/main/scala/com/avast/sst/http4s/server/middleware/CorrelationIdMiddleware.scala

```diff
@@ -58,6 +58,7 @@ object CorrelationIdMiddleware {
 
   final case class CorrelationId(value: String) extends AnyVal
 
+  @SuppressWarnings(Array("scalafix:Disable.toString"))
```

Comment:
Shouldn't we actually use `Show` too instead of suppressing it?

Reply:
In general, I agree, of course. In this instance, this seemed like the better way, because here, a `UUID` is created internally, and immediately converted to `String` via the culprit `toString`. It doesn' seem worth it to create a `Show[UUID]`.

(If the UUID were used in a public API or something like that, that would be a whole different story)